packages/cli/src/lib/Compiler.ts

```diff
@@ -31,15 +28,17 @@ export interface CompilerConfig {
 
 export class Compiler {
   private _manifestDir: string;
+  private _schemaComposer: SchemaComposer;
 
-  constructor(private _config: CompilerConfig) {
+  constructor(private _config: BuildConfig) {
     this._manifestDir = path.dirname(_config.manifestPath);
+    this._schemaComposer = new SchemaComposer(_config);
   }
 
   public async compile(quiet?: boolean, verbose?: boolean): Promise<boolean> {
     try {
       // Load the manifest
-      const manifest = await this._loadManifest();
+      const manifest = await this._schemaComposer.loadManifest();
```

Comment:
In this file, _loadManifest(...) still exists. It is also duplicated in the SchemaComposer. I left a comment in SchemaComposer saying that I think it should be made into its own function to help make these classes smaller.